legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/domain/api/workflow/WorkflowJobAccessContext.java

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.domain.api.workflow;
+
+import org.finos.legend.sdlc.domain.model.workflow.WorkflowJob;
+import org.finos.legend.sdlc.domain.model.workflow.WorkflowJobStatus;
+
+import java.util.List;
+
+public interface WorkflowJobAccessContext
+{
+    WorkflowJob getWorkflowJob(String workflowId, String workflowJobId);
+
+    List<WorkflowJob> getWorkflowJobs(String workflowId, Iterable<WorkflowJobStatus> statuses);
+
+    String getWorkflowJobsLogs(String workflowId, String workflowJobId);
```

Comment:
We're getting one log for one job, so this should be getWorkflowJobLog (no 's' after 'Job' or 'Log').

Reply:
Fixed typo